packages/lexical-editor/README.md

```diff
@@ -0,0 +1,7 @@
+# @webiny/app-lexical-editor
+[![](https://img.shields.io/npm/dw/@webiny/app-page-builder.svg)](https://www.npmjs.com/package/@webiny/app-page-builder) 
+[![](https://img.shields.io/npm/v/@webiny/app-page-builder.svg)](https://www.npmjs.com/package/@webiny/app-page-builder)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
+Enables Webiny Page Builder in the Admin area and on your site. 
```

Comment:
Please update package names:
- `app-page-builder` -> `lexical-editor`
- `app-lexical-editor` -> `lexical-editor`

And add 1 sentence description instead of the existing one :)